DataFormats/CTPPSReco/interface/CTPPSPixelRecHit.h

```diff
@@ -0,0 +1,100 @@
+/*
+ *
+* This is a part of CTPPS offline software.
+* Author:
+*   Fabrizio Ferro (ferro@ge.infn.it)
+*
+*/
+
+#ifndef DataFormats_CTPPSReco_CTPPSPixelRecHit_H
+#define DataFormats_CTPPSReco_CTPPSPixelRecHit_H
+
+
+#include "DataFormats/GeometrySurface/interface/LocalError.h"
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+
+// Reconstructed hits in CTPPS Pixel detector
+
+class CTPPSPixelRecHit{
+
+public:
+
+  CTPPSPixelRecHit(){}
+CTPPSPixelRecHit(LocalPoint lp, LocalError le) : 
```

Comment:
Wouldn't be more convenient (i.e. easier to deal with in case of future modifications of the code) having only one constructor with default paramenters?

Reply:
ok. done.